wrangler-core/src/e2e-test/java/io/cdap/plugin/wrangler/locators/WranglerPropertiesPage.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.wrangler.locators;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+
+
+/**
+ * MySQL source - Properties page - Locators.
+ */
+public class WranglerPropertiesPage {
+    @FindBy(how = How.XPATH, using = "//label[@id='import-pipeline']")
+    public static WebElement importButton;
+
+    @FindBy(how = How.XPATH, using = "//img[@id='resource-center-btn']")
+    public static WebElement plusGreenButton;
+
+    @FindBy(how = How.XPATH, using = "//div[@data-cy='pipeline-metadata']")
+    public static WebElement pipelineName;
+
+    @FindBy(how = How.XPATH, using = "//*[@id=\"pipeline-name-input\"]")
+    public static WebElement pipelineNameIp;
+
+    @FindBy(how = How.XPATH, using = "//*[@data-cy='valium-banner-hydrator']")
+    public static WebElement statusBanner;
+
+    @FindBy(how = How.XPATH, using = "//button[@data-cy='pipeline-metadata-ok-btn']")
+    public static WebElement pipelineSave;
+
+    @FindBy(how = How.XPATH, using = "//div[@data-cy='pipeline-metadata']")
+    public static WebElement appendPipeline;
+
+    public static By renamePipeline() {
+        return By.xpath("//div[@data-cy='pipeline-metadata']");
+    }
```

Comment:
After you make changes in the Actions most of the Locators will be unused here remove them later.

Reply:
Resolved